### ISSUE TITLE
2024_08: BUG: PROD Team Set Up Issues

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#3225]
Identified with @matthewtomo , @dlkibbe, @dlounsbu, @lijenn at 7/22 support, systems_sme, sim_ui workflow sync.

@tejaspesquare We found a variety of bugs and observations associated with team set up in PROD using Chrome, please see below.

- [ ] 1. @matthewtomo Tried to add himself as an administrator to the team we set up today. After he clicked "Add", his name is showing up blank:
![image](https://github.com/user-attachments/assets/3c7e4bd4-077c-4c77-987a-e7aabf676893)

- [ ] 2. We thought this was already in PROD, but looks like it's still in TEST based on issue #https://github.com/lzim/teampsd/issues/3182. We will clarify the business rules with @jamesmrollins.
![image](https://github.com/user-attachments/assets/d581d0bb-0909-4140-9ec6-263a53e13051)

- @matthewtomo then re-set himself up as a Team Lead to complete the team set up and we noticed all his details showed up and nothing is missing:
![image](https://github.com/user-attachments/assets/d676d0f1-0814-4cb1-bd46-97e4d275ca1e)

- [ ] 3. After clicking "Done", @matthewtomo's role reverted back to Administrator and his name disappeared once the team set up was complete:
![image](https://github.com/user-attachments/assets/9b65e914-3140-402b-af33-35fa7c3df67c)

- [ ] 4. We then uploaded a data file for the _team, _shared, and _ind teams from the Administrator Dashboard. We then tried to enter the _team world, but got the "Episodes did not create for _team" error pop-up.
We did not get this error for the _shared and _ind worlds, ONLY the _team world.
![image](https://github.com/user-attachments/assets/9b3102d3-967a-40b5-a3f7-84b42d243412)

FYI: @jamesmrollins @ljmoody

**Comment by matthewtomo** [comment]
**Discussed 7/25 Standup with Dev**
@matthewtomo @tejaspesquare 

- After discussing the "Episodes did not create for _team" error pop-up, we believe this may be related to the default team lead account issues.
- @tejaspesquare was able to manually create the episodes and now the affected team appears to be working as expected in PROD. (Could @dlkibbe and @dlounsbu verify that the team is working as they expect in PROD?)
- @matthewtomo will attempt to recreate issue in current DEV branch to see if it resolved there and ready for TEST branch.

fyi: @jamesmrollins @lijenn @ljmoody

**Comment by matthewtomo** [comment]
**Discussed DEV Standup 7/29/2024**
@matthewtomo @tejaspesquare 

- @matthewtomo will attempt to recreate this issue in current DEV branch to see if it is resolved there and ready for TEST branch.

fyi: @jamesmrollins @lijenn @ljmoody

**Comment by ljmoody** [comment]
@tejaspesquare and @matthewtomo, when creating a team in PROD today (to use as a sample team for mtl.how/demo gif updates), I ran into some inconsistencies with team creation and data files. 

Name of team I created: 123_demo_gifs

- Only _team and _ind worlds are showing up for me (in Settings->Facility Team Setup and the Admin Dashboard select team dropdown)
- When I linked a data file (blue_sample_data_flow.xlsx) to the team, I experienced all of the following at different times
  - I was able to navigate to the Home Page of that team, but the data file was no longer associated with the team
  - I received the "episodes did not create" message in the Admin Dashboard when trying to go to the team's Home Page
  - The data file appeared to be missing from the DDD, but then showed up
  - When I navigated to the Sim, saved an experiment, and returned to the DDD, the values were reset and the data file was no longer listed in Session Start nor Team Data windows

Is it possible that Tejas' working experience in PROD is local only? Wanted to document that I had these same issues today, but don't have screenshots of the experience or console log. I will tomorrow if I bump into the same things.

fyi: @lijenn @jamesmrollins
